dotnet/src/SemanticKernel.Abstractions/AI/ChatClient/ChatMessageExtensions.cs

```diff
@@ -45,6 +52,12 @@ internal static ChatMessageContent ToChatMessageContent(this ChatMessage message
         }
 
         return result;
+
+        Microsoft.Extensions.AI.FunctionCallContent? GetFunctionCallContent(string callId)
+            => response?.Messages
+                .Select(m => m.Contents
+                .FirstOrDefault(c => c is Microsoft.Extensions.AI.FunctionCallContent fcc && fcc.CallId == callId) as Microsoft.Extensions.AI.FunctionCallContent)
```

Comment:
nit: indent this line for readability
```suggestion
                    .FirstOrDefault(c => c is Microsoft.Extensions.AI.FunctionCallContent fcc && fcc.CallId == callId) as Microsoft.Extensions.AI.FunctionCallContent)
```